gamemodes/terrortown/entities/weapons/weapon_ttt_armor.lua

```diff
@@ -0,0 +1,43 @@
+SWEP.Base = "weapon_tttbase"
```

Comment:
Why is there a SWEP weapon?

Reply:
Is there a better way for an armor item that can be bought multiple times?

Reply:
this is a trashy way, then, multiple item bought should be implemented

Reply:
Oof. This would be the best solution. I'll look into the item code.